### ISSUE TITLE
Missing alt Attributes for Images

### ISSUE THREAD

**Issue by SamikshaPIN0981** [SamikshaPIN0981/demoproject#1]
The  tag is missing the alt attribute, which is essential for accessibility

**Comment by SamikshaPIN0981** [comment]
i have added alt attribute in imges
